src/explore-education-statistics-common/src/modules/release/components/ReleaseDataAndFiles.tsx

```diff
@@ -0,0 +1,117 @@
+import Accordion from '@common/components/Accordion';
+import AccordionSection from '@common/components/AccordionSection';
+import Details from '@common/components/Details';
+import { Release } from '@common/services/publicationService';
+import { FileInfo } from '@common/services/types/file';
+import React, { ReactNode } from 'react';
+
+interface Props {
+  renderCreateTablesButton?: ReactNode;
+  onSectionOpenEvent?: (accordionSection: {
+    id: string;
+    title: string;
+  }) => void;
+  release: Release;
+  renderDownloadLink: (file: FileInfo) => ReactNode;
+  renderMetaGuidanceLink: ReactNode;
+  renderPreReleaseAccessLink?: ReactNode;
```

Comment:
I've changed it here, but if there's a rule around props order I think it should be covered by the linter and handled by something like https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md rather than be a manual thing.

Reply:
Great idea :smile: we should put that in with another PR